qiskit_quantuminspire/qi_jobs.py

```diff
@@ -35,15 +59,78 @@ def __init__(
         super().__init__(backend, job_id, **kwargs)
         self._job_ids: List[str] = []
         self._run_input = run_input
+        self.program_name = None
 
-    def submit(self) -> None:
+    async def submit(self) -> None:
         """Submit the (batch)job to the quantum inspire backend.
 
         Use compute-api-client to call the cjm endpoints in the correct order, to submit the jobs.
         """
-        for i in range(1, 3):
-            self._job_ids.append(str(i))
-        self.job_id = "999"  # ID of the submitted batch-job
+        options = self.backend().options
+        # call create algorithm
+        async with ApiClient(config()) as api_client:
```

Comment:
If run_input is a list of quantumcircuits we should be creating a batch job with multiple jobs right?

Reply:
Yes, we would iterate. Have added that code.